app/watch.tsx

```diff
@@ -0,0 +1,30 @@
+import * as React from 'react';
+
+interface WatchProps {
+
+}
+
+interface WatchState {
+  watchRegex: string
+}
+
+class Watch extends React.Component<WatchProps, WatchState> {
+  state = { watchRegex: "*" }
```

Comment:
For now I have so many places where I might've forgotten adding semicolons. will add that to a linter and fix all at once out of this PR.